### PR TITLE
fix setUpClass in remote execution CLI tests

### PR DESCRIPTION
**Pull request by pondrejk**
Related to #4227

### CODE REVIEW

File: tests/foreman/cli/test_remoteexecution.py [hunk 1]
```diff
@@ -176,24 +191,58 @@ class RemoteExecutionTestCase(CLITestCase):
...
+    @skip_if_not_set('clients', 'fake_manifest')
     def setUpClass(cls):
-        """Create an organization which can be re-used in tests."""
+        """Create Org, Lifecycle Environment, Content View, Activation key
+
+        """
         super(RemoteExecutionTestCase, cls).setUpClass()
-        cls.organization = make_org()
+        # Create new org, environment, CV and activation key
+        RemoteExecutionTestCase.org = make_org()
```
Comment (svtkachenko) [inline tests/foreman/cli/test_remoteexecution.py]: Why `cls` was changed to actual class name?

**Comment by rplevka** [comment]
please, provide some test run results

File: tests/foreman/cli/test_remoteexecution.py [hunk 2]
```diff
@@ -176,24 +191,58 @@ class RemoteExecutionTestCase(CLITestCase):
     """Implements job execution tests in CLI."""
 
     @classmethod
+    @skip_if_not_set('clients', 'fake_manifest')
     def setUpClass(cls):
-        """Create an organization which can be re-used in tests."""
+        """Create Org, Lifecycle Environment, Content View, Activation key
+
```
Comment (oshtaier) [inline tests/foreman/cli/test_remoteexecution.py]: Redundant line

File: tests/foreman/cli/test_remoteexecution.py [hunk 3]
```diff
@@ -176,24 +191,58 @@ class RemoteExecutionTestCase(CLITestCase):
...
+        """
         super(RemoteExecutionTestCase, cls).setUpClass()
-        cls.organization = make_org()
+        # Create new org, environment, CV and activation key
```
Comment (oshtaier) [inline tests/foreman/cli/test_remoteexecution.py]: Seems as redundant comment as it provides same information as docstring

File: tests/foreman/cli/test_remoteexecution.py [hunk 4]
```diff
@@ -176,24 +191,58 @@ class RemoteExecutionTestCase(CLITestCase):
...
+            u'organization-id': RemoteExecutionTestCase.org['id'],
+            u'content-view-id': RemoteExecutionTestCase.content_view['id'],
+            u'lifecycle-environment-id': RemoteExecutionTestCase.env['id'],
+            u'activationkey-id': RemoteExecutionTestCase.activation_key['id'],
+        })
+
+    def setUp(self):
+        """Create VM, subscribe it to satellite-tools repo, install katello-ca
+        and katello-agent packages, add remote execution key
+
```
Comment (oshtaier) [inline tests/foreman/cli/test_remoteexecution.py]: Redundant line

**Comment by oshtaier** [comment]
ACK pending nitpicks and @abalakh review

File: tests/foreman/cli/test_remoteexecution.py [hunk 5]
```diff
@@ -176,24 +191,58 @@ class RemoteExecutionTestCase(CLITestCase):
...
+        self.client.install_katello_ca()
+        # Register content host, install katello-agent
+        self.client.register_contenthost(
+            RemoteExecutionTestCase.org['label'],
+            RemoteExecutionTestCase.activation_key['name'],
         )
-        cls.client.enable_repo(REPOS['rhst7']['id'])
-        cls.client.install_katello_agent()
-        add_remote_execution_ssh_key(cls.client.hostname)
+        self.host = Host.info({'name': self.client.hostname})
```
Comment (abalakh) [inline tests/foreman/cli/test_remoteexecution.py]: Can't see any usage of this variable. Isn't it redundant?

**Comment by pondrejk** [comment]
Dear reviewers, please find the test results in the standalone automation job #268. Cheers!

**Comment by ldjebran** [comment]
as far as know I'am for the the moment blocked by a bug https://bugzilla.redhat.com/show_bug.cgi?id=1406315 affecting sat 6.3 that cannot install katello agent or any rpm from the subscribed sat , are you using a new compose?

**Comment by pondrejk** [comment]
@ldjebran @oshtaier @abalakh @rochacbruno  do you think we can merge it anyway? Since we know about the rpm sync issue, and the test passes in 6.2. The thing is that this is blocking the cherry-pick of some other changes to remoteexecution.py. What do you guys think?

**Comment by rochacbruno** [comment]
@pondrejk @oshtaier @abalakh I am ok merging this and opening an issue regarding the rpm sync issue if needed. (we must keep cherry picks in sync)

**Comment by rochacbruno** [comment]
@ldjebran please check if your settings work with this, and anyway if you are ok, lets merge this and fill issue to keep track of the problem.

**Comment by ldjebran** [comment]
all comments was addressed, run one test to be sure  http://pastebin.test.redhat.com/463531

**Comment by ldjebran** [comment]
tests made with sat Settings content default_download_policy immediate to skip the bug 1406315

```console 
platform linux2 -- Python 2.7.13, pytest-3.0.6, py-1.4.32, pluggy-0.4.0 -- /home/dlezz/.pyenv/versions/2.7.13/envs/sat-6.3.0/bin/python2.7
cachedir: .cache
rootdir: /home/dlezz/projects/robottelo-fork, inifile: 
plugins: xdist-1.15.0, cov-2.4.0
collected 9 items 
2017-03-13 11:45:54 - conftest - DEBUG - Found WONTFIX in decorated tests ['1269196', '1245334', '1217635', '1226425', '1156555', '1204686', '1267224', '1103157', '1230902', '1214312', '1079482']

2017-03-13 11:45:54 - conftest - DEBUG - Collected 9 test cases


tests/foreman/cli/test_remoteexecution.py::RemoteExecutionTestCase::test_positive_run_custom_job_template PASSED
tests/foreman/cli/test_remoteexecution.py::RemoteExecutionTestCase::test_positive_run_default_job_template PASSED
tests/foreman/cli/test_remoteexecution.py::RemoteExecutionTestCase::test_positive_run_scheduled_job_template PASSED

======================================================= 6 tests deselected ========================================================
============================================ 3 passed, 6 deselected in 656.78 seconds =============================================
(sat-6.3.0) dlezz@elysion:~/projects/robottelo-fork$ 
```

**Pull request merged**